zeek_log_transport.sh

```diff
@@ -237,14 +347,25 @@ status "Sending logs to rita/aihunter server $aih_location , My name: $my_id , l
 status "Preparing remote directories"
 ssh $extra_ssh_params "$aih_location" "mkdir -p ${remote_top_dir}/$today/ ${remote_top_dir}/$yesterday/ ${remote_top_dir}/$twoda/ ${remote_top_dir}/$threeda/ ${remote_top_dir}/current/"
 
+recv_loc="`pwd`/zeek-log-transport.yaml"
+request_logs=`get_send_logs $aih_location "/etc/AI-Hunter/zeek-log-transport.yaml" "$recv_loc"`
```

Comment:
There are cases where we will want different retention and logs from different sensors. If the sensor is small enough we can transfer all the logs and keep them for longer, but another sensor's logs might be too big where we need to limit what is sent and only keep it for a limited time.

I can think of a couple ways to do this:
- We could pull from a sensor-specific location first (e.g. `$remote_top_dir/zeek-log-transport.yaml`) and fall back to `/etc/AI-Hunter/zeek-log-transport.yaml` if that fails. This seems the most straightforward to me.
- We could put sensor-specific config sections in the same `/etc/AI-Hunter/zeek-log-transport.yaml` file. This would be easier if we were actually parsing structured YAML somehow, but it might be too difficult with the current method of using grep.